src/scrape/scrape_job_test.go

```diff
@@ -200,6 +233,40 @@ func TestPopulateK8s(t *testing.T) {
 	}
 }
 
+func TestRestartCountDeltaValues(t *testing.T) {
+	intgr, err := integration.New("test", "test", integration.InMemoryStore())
+	assert.NoError(t, err)
+	intgr.Clear()
```

Comment:
why `intgr.Clear()` is needed?

Reply:
As it is (or it should be) a brand new integration, I guess it is not needed. I'm digging into it in this and the populate-test we already had. Thanks!

Reply:
Removed here: [bc62e84](https://github.com/newrelic/nri-kubernetes/pull/382/commits/bc62e845386de80d99d1d5b928fc25d5b72ecd98)